cypress/integration/registro.spec.js

```diff
@@ -0,0 +1,56 @@
+import { VAR, URL } from '../const/constants';
+import { IDs } from '../page-objects/registro';
+// / <reference types="cypress" />
+
+context('Register', () => {
+  beforeEach(() => {
+    cy.visit(URL.index);
+    cy.get(`.${IDs.idSingIn}`).click();
```

Comment:
Also, the same example apply for getting the fields, and don't rewrite code:
instead of `cy.get(`#${IDs.emailCreate}`).type(VAR.textSmall)` you can do `addEmail(VAR.textSmall)`
where `addEmail` is:
```
  addEmail(text) {
    cy.get(`#${IDs.emailCreate}`).type(text);
. . .
  }

Reply:
@josepplloo  te parece que los cargue en el archivo de registro.js ??